docs/guides/overview.md

```diff
@@ -352,11 +352,36 @@ Important Note About Dynamic Segments
 If you have dynamic segments in your URL, a transition from `/users/123`
 to `/users/456` does not call `getInitialState`, `componentWillMount`, `componentWillUnmount` or `componentDidMount`. If you are using those lifecycle hooks to fetch
 data and set state, you will also need to implement
-`componentWillReceiveProps` on your handler, just like any other
+`componentWillReceiveProps` on your handler, and its child components, just like any other
```

Comment:
I'd correct that to be “and its stateful children”. Stateless children don't need to implement anything.
Also nitpicking: probably don't need a comma before “and”.